samples/apps/pipes.jakt

```diff
@@ -0,0 +1,183 @@
+
+enum Stream : c_int {
+    STDIN  = 0,
+    STDOUT = 1,
+    STDERR = 2
+}
+
+struct Pipe {
+    reading_end: c_int
+    writing_end: c_int
+}
+
+class System {
+    public function getErrno() -> c_int {
```

Comment:
Jakt follows a similar coding style as serenity, and serenity uses `snake_case` for methods and omits `get_` from getters.

Reply:
Function name cant be just `errno` since its defined as a macro. Will rename to `syscall_error_number`